release.md

```diff
@@ -1,19 +1,22 @@
-# Release procedure
+# Release procedures
 
-## Prerequisites
-1. Stop merges to `master` branch except for the PR in the next section.
+## Creating a release branch
 
-## Creating a new release branch
-1. Create a release branch on the upstream.
-    ```
-    git fetch upstream master
-    git push upstream upstream/master:refs/heads/vX.Y
-    ```
+### Mark `master` as the next release
+Moving `master` to the next release is done by creating a new `vX.Y+1.Z-alpha.0` tag for the next version. It can be either just the `git` tag or a proper `alpha` release, but given the very little difference the `beta` of the previous release, it's usability is limited to worth the effort.
```

Comment:
Procedure should explain what we do, not leave the decision for someone who executes it. We either do alpha tags or not. 
```suggestion
Moving `master` to the next release is done by creating a new `vX.Y+1.Z-alpha.0` tag for the next version.
```

Reply:
removed